components/AboutInfo.js

```diff
@@ -0,0 +1,82 @@
+        <div className="lg:w-4/5 w-11/12  bg-gray-900 rounded-lg px-4  py-2  my-10 m-auto text-center">
+            <h1 className="text-5xl text-yellow-500">Our Mission</h1>
+            <div className="lg:flex justify-between items-start">
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto  bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_edu.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75  rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-green-600 text-black text-opacity-100 uppercase">Education</h3>
+                        <p className="mx-5 pb-5">
+                        GDSC CMRIT provides a platform for the growth and advancement of the student community in the field of technology by providing peer-to-peer learning environment by conducting , webinars and events , providing hands on experiences on topics like Web Development, Mobile App Development, Data Science, Blockchain,  Machine Learning, Contributing to Open Source and much more!</p>
+                    </div>
+                    
+                </div>
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_collab.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75 rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-green-600 text-black text-opacity-100 uppercase"> Collaboration</h3>
+                        <p className="mx-5 pb-5 ">
+                        We are not just a community but a mini India where people from diverse backgrounds and different majors come together to collaborate and co-create new innovative ideas that would help solve real world problems, build solutions for the local problems and beget a community that aspires for a positive and impactful change.</p>
+                    </div>
+                    
+
+                </div>
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_innovate.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75 rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-black text-opacity-100 uppercase"> Innovations</h3>
+                        <p className=" mx-5 pb-5">
+                        Through education and collaboration we want to provide students a platform where they can innovate and experiment along with us. Developing innovative products is what separates one from the crowd. Therefore, we want to lay the foundation necessary to bring our innovative visions to life through the power of technology.</p>
+                    </div>
+                    
+                </div>
+            </div>
+        </div>
```

Comment:
and again same with this. all these three can be taken out into separate components.